src/VideoReceiver/gstqgcvideosinkbin.c

```diff
@@ -311,12 +311,12 @@ _vsb_get_type(void)
             sizeof(GstQgcVideoSinkBinClass),
             NULL,
             NULL,
-            (GClassInitFunc)_vsb_class_init,
+            (GClassInitFunc)(void *)_vsb_class_init,
```

Comment:
I would rather change the method API to `void *, void *` so we don't have to cast here, and do the necessary cast in the method.

Reply:
Can you be a bit specific as to what method you are referring to?

Are you referring to `g_type_register_static`? If so I don't think we can since that is part of a dependency and not code that we have written ourselves.

Reply:
The `_vsb_class_init` method. Both arguments are expected to be `void*` pointers.

Reply:
@bkueng for `_vsb_class_init` I believe it only has one argument right?

I agree though, I have changed the method API to ingest a void * rather than doing a preliminary cast before calling the method.

Reply:
Yes you need to add a 2. argument, `void* classData`. Then use `Q_UNUSED(classData);` in the method to suppress the unused warning.

Reply:
Ah til, `Q_UNUSED`. Thanks for the pointers, @bkueng.

I have added your suggestions and this PR should be good to go now.